src/main/kotlin/ch/hevs/cloudio/cloud/restapi/controllers/UserAccessControlController.kt

```diff
@@ -0,0 +1,91 @@
+package ch.hevs.cloudio.cloud.restapi.controllers
+
+import ch.hevs.cloudio.cloud.apiutils.*
+import ch.hevs.cloudio.cloud.model.Authority
+import ch.hevs.cloudio.cloud.model.PrioritizedPermission
+import ch.hevs.cloudio.cloud.repo.authentication.User
+import ch.hevs.cloudio.cloud.repo.authentication.UserRepository
+import ch.hevs.cloudio.cloud.restapi.CloudioBadRequestException
+import ch.hevs.cloudio.cloud.restapi.CloudioForbiddenException
+import ch.hevs.cloudio.cloud.restapi.CloudioOkException
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class UserAccessControlController(var userRepository: UserRepository) {
+
+    @RequestMapping("/api/v1/getUserAccessRight", method = [RequestMethod.GET])
```

Comment:
You could pass the base URI as parameter to the controller itself and then use relative URIs at method level. It would just simplify the code and ensure uniformity.

Reply:
I'll have a look at that!